STYLEGUIDE.md

```diff
@@ -1,39 +1,311 @@
 
-## Write for translations
+### 1.2 Hyphens? Spaces? Neither!
 
-* Avoid gerunds (-ing words)
+Tech has created many new words by associating two words with each other like "front end" and "web site." On a long enough timeline, these migrate to hyphenated forms: "web site" became "web-site." And finally they become concatenated words: "website." We use the concatenated forms.
 
-## Write for code
+#### Do
 
-* Use monospace fonts for code, commands, file names, copy-ables; use bold for menus, URLs (clickable), and other UI copy
-* Use carets to help users navigate menus; use “select” instead of “click”
-* Write compliant JSON: no single quotes, no comments
-* Use 2-space indents for code blocks
+> Frontend and fullstack developers often build websites with React.
 
-## Write for Clerk
+#### Don't 
 
-* Authentications state (signed-in, signed-out, unknown) should be hyphenated and bolded to distinguish “recognized Clerk states” from general states of being (”She was signed out of the app” vs “When the user is signed-out…”).
\ No newline at end of file
+> Front-end and full stack developers often build web-sites with React.
+
+### 1.3 When mentioning a documented component, function, etc, multiple times on a page, link to the reference documentation on the first mention of that item.
+
+#### Do
+
+> The [`currentUser()`](https://clerk.com/docs/references/nextjs/current-user) helper will return the [`User`](https://clerk.com/docs/references/javascript/user/user) object of the currently active user. The following example userse the `currentUser()` helper to access the `User` object for the authenticated user.
+
+#### Don't 
+
+> The [`currentUser()`](https://clerk.com/docs/references/nextjs/current-user) helper will return the [`User`](https://clerk.com/docs/references/javascript/user/user) object of the currently active user. The following example userse the [`currentUser()`](https://clerk.com/docs/references/nextjs/current-user) helper to access the [`User`](https://clerk.com/docs/references/javascript/user/user) object for the authenticated user.
+
+## 2. Write for clarity
+
+### 2.1 “We” are the authors; “you” is the reader.
+
+Clerk is the entity writing these docs and providing these services. The learner is the person consuming these things and building their own projects. To distinguish between the two and write with clarity, we write from Clerk's perspective, using "we/us/our/ours," and refer to the learner with "you/your/yours."
+
+#### Do
+
+> Our `<ClerkProvider />` provides active session and user context to Clerk's hooks and other components. Import it into your app by adding `import { ClerkProvider } from '@clerk/nextjs'` at the top of your file.
+
+#### Don't 
+
+> `<ClerkProvider />` provides active session and user context to our hooks and other components. Let's import it into our app by adding `import { ClerkProvider } from '@clerk/nextjs'` at the top of the file.
```

Comment:
Would like @BRKalow input on this - we had discussed the use of "we/our/ours" should not be used in the docs, and instead use objective language when referring to Clerk.
Instead of "Our `<ClerkProvider />`", it would be written "Clerk's `<ClerkProvider/ >`".

Reply:
I'm so glad you've been thinking about this! Want to get alignment before writing this in stone.

Reply:
I feel like we/us/our/ours works for blog posts, Twitter and things like that, while the Docs should be more 'Clerk.' I just took a look at the Next.js and Prisma sites and they follow that convention --- we/us/our/ours in the Blog Pots and Next.js/Prisma in the docs. That said, I think either works in the docs and its more a matter of revising to make that consistent.

Reply:
That's right, up to this point we've tried to use `Clerk` instead of the mentioned pronouns. I personally find it a more effective writing style for reference-like material. Like @royanger said, we/us/our/ours feels more appropriate in blog posts and other mediums where there is generally a clear individual author listed.

Ultimately I think consistency should win here, so whatever recommendation we go with is okay with me as long as we're consistent. 👍

Reply:
This styleguide seeks to pave cowpaths rather than enforce new voice and tone. If this pattern were harmful, we might change it, but it makes sense, and it sounds like our docs already use it. Let's make this the official recommendation and canonize it in the styleguide